src/LocationMarker.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from "react";
```

Comment:
Since we already have `/components` folder, wouldn't it be better to move `src/LocationMarker.js` and `src/Map.js` inside it?

Reply:
Yes, it would. We want to keep the project well-organized, so all components should be in the `/components` folder.

Reply:
A separate comment: it's actually not necessary to import `React` along with `useEffect`. I know the docs for hooks show this way of importing, but unless you're using the `React` variable it can be safely removed.

Reply:
rebased the branch according to the main branch and moved the files with respect to the functionality according to the folder structure